firebase.json

```
@@ -155,6 +155,7 @@
       { "source": "/ui/material3-updates", "destination": "/ui/material", "type": 301 },
       { "source": "/ui/splash-screen", "destination": "/platform-integration/android/splash-screen", "type": 301 },
       { "source": "/ui/splash-screen/android-splash-screen", "destination": "/platform-integration/android/splash-screen", "type": 301 },
+      { "source": "/platform-integration/ios/splash-screen", "destination": "/platform-integration/ios/launch-screen.md", "type": 301 },
```

Don't specify the ".md", as that refers to source, not the actual URL. This is breaking the build. 

```suggestion
      { "source": "/platform-integration/ios/splash-screen", "destination": "/platform-integration/ios/launch-screen", "type": 301 },
```